file-generating/ronn/README.md

```diff
@@ -0,0 +1,162 @@
+# Ronn
+
+Ronn converts markdown files to manpages, and HTML manpages for the web.
+
+Ronn is based off Ruby, and depending on your system, it could be easily installed.<br>
+But what if you don't have Ronn package for your system (and don't have/want ruby installed), or you don't have root access,<br>
+then here's a good example for a container use.
+
+<br>
+
+Jump to:
+ - [Downloading the container](#downloading-the-container).
+ - [Adding the container to your PATH](#adding-the-container-to-your-path).
+ - [Usage](#usage).
+ - [More examples](#more-examples).
+ - [Specifying the output file](#specifying-the-output-file).
+ - [Making your own manpage](#making-your-own-manpage).
+ - [Demo](#demo).
+
+
+<br>
+
+#### What you need:
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim`, or `nano`.
+ - A markdown test file, like [this one](https://raw.githubusercontent.com/sylabs/examples/master/file-generating/ronn/manpage-test.md).
+
+
+___
+
+<br>
+
+
+### Downloading the container:
+
+For `x86_64`:
+
+```
+$ singularity pull ronn.sif library://sylabs/examples/ronn:latest
+```
+
+For `armv7l`: (`32bit`)
+
+```
+$ singularity pull ronn.sif library://sylabs/examples/ronn-arm:latest
+```
+
+<br>
+
+Verifying the container:
+
+```
+$ singularity verify ronn.sif
+Verifying image: ronn.sif
+Data integrity checked, authentic and signed by:
+        Sylabs Admin <support@sylabs.io>, KeyID EDECE4F3F38D871E
+```
+
+<br>
+
+### Adding the container to your PATH:
+
+We are going to do this without any root access, so we will install `ronn.sif` to `~/.local/bin/`:<br>
+You can skip this step if you already have `${HOME}/.local/bin/`.<br>
+Of course you can always move it to `/usr/local/bin/`, but that requires root access.
+
+First, make the `${HOME}/.local/bin/`:
+
+```
+$ mkdir -p ${HOME}/.local/bin/
+```
+
+Then, add it to your PATH
+
+```
+$ echo 'export PATH=${PATH}:${HOME}/.local/bin/' >> ~/.bashrc
+$ source ~/.bashrc
+```
+
+<br>
+
+Finaly, move `ronn.sif` to `~/.local/bin/`
+
+```
+$ mv ronn.sif ~/.local/bin/
+```
+
+<br>
+
+## Usage:
+
+For this example, let's make a manpage for this container.
+
+First, download the markdown file:
+
+```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/file-generating/ronn/ronn.sif.md
```

Comment:
check this url as well

Reply:
The url seems fine... once it is merged into sylabs.